app/Process.php

```diff
@@ -162,19 +189,62 @@ protected function run(string $command) : int
         return proc_close($proc);
     }
 
+    public function getOutputBuffer(): string
+    {
+        return $this->getFileContents($this->outputFileName);
+    }
+
+    public function getErrorBuffer(): string
+    {
+        return $this->getFileContents($this->errorFileName);
+    }
+
+    private function getFileContents(string $filename): string
+    {
+        $output = '';
+        $handle = @fopen($filename, 'r');
+
+        while (($buffer = fgets($handle)) !== false) {
+            $output .= trim($buffer) . "\n";
+        }
+
+        if (! feof($handle)) {
+            $output = 'Erro: falha inexperada na leitura do arquivo!';
+        }
+
+        return $output;
+    }
+
     protected function getDescriptors() : array
     {
         if ($this->output || env('FWD_VERBOSE')) {
             return [STDIN, STDOUT, STDERR];
         }
 
-        $devNull = fopen('/dev/null', 'w');
-
-        return [STDIN, $devNull, $devNull];
+        $this->outputFileName = @tempnam(sys_get_temp_dir(), 'fwd_output_');
```

Comment:
would be good to test this on the phar version, run:

`./fwd app:build` and use the `builds/fwd` bin for testing